.gitignore

```diff
@@ -3,4 +3,5 @@
 .DS_Store
 
 /node_modules/
+/test/bats/
```

Comment:
@alnikyur why?

Reply:
@tyranron  I was added test/bats/ directory to ignoring rules, because when i upgrade tools in docker images i did local tests, and after this must to delete this directory. If this changes if not necessary i will remove it.

Reply:
@alnikyur you have outdated toolchain. See [this commit](https://github.com/instrumentisto/gitlab-builder-docker-image/commit/71c663390f81ae43c3cfacdb2cf47f3e937ef9f2) for changes. You should not ignore Git log, otherwise you won't understand what is happening.

Reply:
@alnikyur revert that